code/modules/jobs/job_types/enclave.dm

```diff
@@ -336,51 +275,27 @@
 	outfit = /datum/outfit/job/enclave/peacekeeper/f13specialist
 	exp_requirements = 800
 
-	loadout_options = list(
-		/datum/outfit/loadout/combatmedic, // Medical Equipment
-		/datum/outfit/loadout/combatengie, // Grenade Launcher
-		)
-
 /datum/outfit/job/enclave/peacekeeper/f13specialist
 	name = "Enclave Specialist"
 	jobtype = /datum/job/enclave/f13specialist
 	suit = /obj/item/clothing/suit/armor/bulletproof
-	suit_store = /obj/item/gun/ballistic/automatic/smg/mp5
+	suit_store = /obj/item/gun/ballistic/automatic/r93
```

Comment:
desc = "A lightweight assault rifle manufactured by the Brotherhood of Steel with a folding stock, based on weapons from the R-series platforms. It is generally issued to Brotherhood Knights for scouting missions."

The description of the R93. Doesn't feel like an Enclave gun.